ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/line-display.tsx

```diff
@@ -144,13 +153,21 @@ export const drawLine = ({
   const color = model.get('color')
   const iconStyle = new ol.style.Style({
     stroke: new ol.style.Stroke({
-      color: color ? color : '#914500',
-      width: 3,
+      color: isInteractive
+        ? contrastingColor
+        : color
+        ? color
+        : contrastingColor,
+      width: isInteractive ? 6 : 4,
     }),
   })
   const drawnLineIconStyle = new ol.style.Style({
     stroke: new ol.style.Stroke({
-      color: color ? color : '#914500',
+      color: isInteractive
+        ? contrastingColor
+        : color
+        ? color
+        : contrastingColor,
       width: 2,
```

Comment:
These are the dashes that show up when the buffer is large enough. When I make them wider, if the buffer is not large enough to be seen, the line becomes harder to click on because the dashes are drawn over the line, but they can't been seen.